### PR TITLE
Add ISY994 Climate Documenation

### PR DESCRIPTION
**Pull request by shbatm**
**Description:**
Add documentation for the ISY994 component Climate integration.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#20970

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/8475"></a>

### CODE REVIEW

File: source/_components/climate.isy994.markdown [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
```
Comment (klaasnicolaas) [inline source/_components/climate.isy994.markdown]: You can add the information on this page to the component page, delete this one.

File: source/_components/isy994.markdown [hunk 2]
```diff
@@ -35,6 +36,7 @@ The ISY994 controller is manufactured by [Universal Devices](https://www.univers
 There is currently support for the following device types within Home Assistant:
 
 - Binary Sensor
+- [Climate](/components/climate.isy994/)
```
Comment (klaasnicolaas) [inline source/_components/isy994.markdown]:
```suggestion
- Climate
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by shbatm** [comment]
Changes have been made. Please take a look.

**Review by klaasnicolaas — APPROVED** [review]

**Review by klaasnicolaas — DISMISSED** [review]

**Comment by klaasnicolaas** [comment]
✅ Can be merged as soon the parent PR got merged.

**Comment by shbatm** [comment]
Status update. Still pending PyPi module update from another developer and still pending code acceptance on back end.

**Comment by klaasnicolaas** [comment]
Parent PR still open

**Comment by shbatm** [comment]
Force-push is a re-base only. No changes made.

File: source/_components/isy994.markdown [hunk 3]
```diff
@@ -15,6 +16,7 @@ ha_release: 0.28
 ha_iot_class: Local Push
 redirect_from:
   - /components/binary_sensor.isy994/
+  - /components/climate.isy994/
```
Comment (klaasnicolaas) [inline source/_components/isy994.markdown]:
```suggestion
  - /components/climate.isy994/
```

Please remove this line

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by shbatm** [comment]
Updated with additional changes to be included in home-assistant/home-assistant#20970

File: source/_components/isy994.markdown [hunk 4]
```diff
@@ -207,3 +213,56 @@ The *actions* program indicates what should be performed for the following devic
...
+      - id: 5
+        type: 2
+
+```
+
+{% configuration %}
+id:
+  description: The ISY Variable ID number to use.
+  required: true
+  type: int
```
Comment (frenck) [inline source/_components/isy994.markdown]:
```suggestion
  type: integer
```

File: source/_components/isy994.markdown [hunk 5]
```diff
@@ -207,3 +213,56 @@ The *actions* program indicates what should be performed for the following devic
...
+type:
+  description: The ISY Variable Type (1=Integer, 2=State).
+  requried: true
+  type: int
```
Comment (frenck) [inline source/_components/isy994.markdown]:
```suggestion
  type: integer
```

File: source/_components/isy994.markdown [hunk 6]
```diff
@@ -207,3 +213,56 @@ The *actions* program indicates what should be performed for the following devic
...
+  type: string
+payload_on:
+  description: The payload that represents the on state (`binary_sensor` and `switch` only).
+  required: false
+  type: int
+  default: 1
+payload_off:
+  description: The payload that represents the off state  (`binary_sensor` and `switch` only).
+  required: false
+  type: int
```
Comment (frenck) [inline source/_components/isy994.markdown]:
```suggestion
  type: integer
```

File: source/_components/isy994.markdown [hunk 7]
```diff
@@ -82,6 +84,10 @@ tls:
   description: This entry should reflect the version of TLS that the ISY controller is using for HTTPS encryption. This value can be either 1.1 or 1.2. If this value is not set, it is assumed to be version 1.1. This is the default for most users. ISY994 Pro users may likely be using 1.2. When using HTTPS in the host entry, it is best practice to set this value.
   required: false
   type: string
+isy_variables:
+  description: ISY Variables can be explicitly defined as as sensors, binary sensors, or switches.  See [Variables as Devices](#variables-as-devices) below.
+  required: false
+  type: dict
```
Comment (frenck) [inline source/_components/isy994.markdown]: `dict` is not a valid configuration type for the documentation.

**Review by frenck — CHANGES_REQUESTED** [review]
Hi there @shbatm, I've looked at your PR and left some comments. Could you please take a look? 👍

File: source/_components/isy994.markdown (hunk 7 above)
Comment (shbatm) [reply source/_components/isy994.markdown]: changed to (map) as per https://www.home-assistant.io/components/sensor.mqtt/ `device`

**Comment by frenck** [comment]
Closing PR, since the parent PR has been closed.

**Pull request closed without merging**